crates/eth-rpc/tests/integration.rs

```diff
@@ -1,22 +1,24 @@
 #[cfg(test)]
 mod integration_tests {
     use std::str::FromStr;
-    use std::sync::Arc;
 
+    use dotenv::dotenv;
     use ethers::prelude::{Block as EthersBlock, Http as EthersHttp, H256 as EthersH256};
-    use kakarot_rpc::test_utils::setup_kakarot_rpc_integration_env;
-    use kakarot_rpc_core::test_utils::deploy_helpers::construct_kakarot_test_sequencer;
+    use kakarot_rpc::test_utils::start_kakarot_rpc_server;
+    use kakarot_rpc_core::test_utils::deploy_helpers::KakarotTestEnvironmentContext;
+    use kakarot_rpc_core::test_utils::fixtures::kakarot_test_env_ctx;
     use reth_primitives::U64;
+    use rstest::*;
 
+    #[rstest]
     #[tokio::test]
-    async fn test_get_block_rpc() {
-        // Setup the test sequencer
-        let starknet_test_sequencer = Arc::new(construct_kakarot_test_sequencer().await);
+    async fn test_get_block_rpc(kakarot_test_env_ctx: KakarotTestEnvironmentContext) {
+        // Load env
+        dotenv().ok();
```

Comment:
do we need to change anything in our .env?
if so that would also affect the docker compose all-in-one that we have make katana-up, i think

Reply:
No need to change anything, actually I had to add this because `dotenv()` used to be called by `deploy_kakarot_system` which was called in `setup_kakarot_rpc_integration_env`. Since I refactored `setup_kakarot_rpc_integration_env` to avoid all the deployment, the .env wasn't loading anymore which caused an issue.

Reply:
great